### PR TITLE
Update subham.json for hosting github pages

### PR DESCRIPTION
**Pull request by skuma251**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by iostpa — CHANGES_REQUESTED** [review]
you forgot to add a comma after line 5, please add that

**Comment by skuma251** [comment]
> you forgot to add a comma after line 5, please add that

added

**Review by iostpa — APPROVED** [review]

**Review by iostpa — APPROVED** [review]

**Comment by CuteDog5695** [comment]
im a helper btw, not maintainer. you'd need to wait for an maintainer to merge it. helpers cant merge pr's

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**
